sycl/CMakeLists.txt

```diff
@@ -137,6 +138,10 @@ COMMENT "Copying SYCL headers ...")
 
 # Configure SYCL headers
 install(DIRECTORY "${sycl_inc_dir}/." DESTINATION "${LLVM_INST_INC_DIRECTORY}" COMPONENT sycl-headers)
+install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/xmethods"
+  DESTINATION ${LLVM_PYTHON_DIRECTORY}
+  COMPONENT sycl-headers
```

Comment:
Can we have separate component for this file?

Reply:
Added `sycl-headers-extras`. Please suggest a different name if you prefer one.